packages/smart-contracts/hardhat.config.ts

```diff
@@ -365,13 +350,14 @@ task(
   await deployWithCreate2FromList(hre as HardhatRuntimeEnvironmentExtended);
 });
 
-task(
-  'update-contracts',
-  'Update the latest deployed contracts from the Create2DeploymentList',
-).setAction(async (_args, hre) => {
-  await hre.run(DEPLOYER_KEY_GUARD);
-  await updateContractsFromList(hre as HardhatRuntimeEnvironmentExtended);
-});
+task('update-contracts', 'Update the latest deployed contracts from the Create2DeploymentList')
+  .addFlag('safe', 'Is the update to be performed in Safe context')
+  .setAction(async (args, hre) => {
+    const safeMode = args.safe ?? false;
+    console.log(safeMode);
```

Comment:
```suggestion
```

Reply:
I find the name `safeMode` very confusing. It's most often used to describe a type of execution, but here it's used to describe an actual mode that's using Safe, if I'm not mistaken

Reply:
Agreed

Reply:
Maybe `isSafeMultisig` would make it more clear?

Reply:
Also consider clarifying all of the header comments: 

```
safeMode Is the transaction to be executed by a Safe Multisig
```